ic-canister-client/src/error.rs

```diff
@@ -1,4 +1,6 @@
 use ic_exports::ic_cdk::api::call::RejectionCode;
+#[cfg(feature = "state-machine-tests-client")]
+use ic_exports::ic_test_state_machine::UserError;
```

Comment:
please do not use conditional `use`, instead, use the full qualified import where UserError is used

Reply:
Done.
Is it part of our code style rules?

Reply:
no, it's only an attempt to reduce the number of `#[cfg(feature = "???")]` that sprout like mushrooms on a rainy day ;)